packages/liferay-npm-scripts/src/utils/run.js

```diff
@@ -9,7 +9,7 @@ const {spawn} = require('cross-spawn');
  * Convenience helper for running commands.
  */
 function run(executable, ...args) {
-	const command = `${executable} ${args.join(' ')}`;
+	const command = [executable, ...args].join(' ');
```

Comment:
In adding tests, I saw that we were reporting a failed command like `false` with a trailing space, so this change fixes that.